packages/web-components/src/components/structured-list/__stories__/structured-list.stories.ts

```diff
@@ -0,0 +1,272 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { html } from 'lit-element';
+import readme from './README.stories.mdx';
+import '../index';
+import 'carbon-web-components/es/components/tooltip/index';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+
+export const Default = ({ parameters }) => {
+  const { colSpan1, colSpan2, colSpan3, colSpan4 } = parameters?.props?.StructuredList ?? {};
+  return html`
+    <dds-structured-list
+      col-span-1="${colSpan1 ?? ''}"
+      col-span-2="${colSpan2 ?? ''}"
+      col-span-3="${colSpan3 ?? ''}"
+      col-span-4="${colSpan4 ?? ''}"
```

Comment:
```suggestion
      col-span-1="${ifNonNull(colSpan1)}"
      col-span-2="${ifNonNull(colSpan2)}"
      col-span-3="${ifNonNull(colSpan3)}"
      col-span-4="${ifNonNull(colSpan4)}"
```

Suggest this change for all variants.